src/main/java/com/syncapse/jenkinsci/plugins/awscloudformationwrapper/CloudFormation.java

```diff
@@ -372,7 +374,18 @@ private Stack waitForStackToBeCreated() throws TimeoutException {
     private void printStackEvents() {
         DescribeStackEventsRequest r = new DescribeStackEventsRequest();
         r.withStackName(getExpandedStackName());
-        DescribeStackEventsResult describeStackEvents = amazonClient.describeStackEvents(r);
+        int count = 0;
```

Comment:
I think count should be set to 1 here, so that the initial retry is not:
'''
Got an error getting stack status, retrying 0 out of 3 times
'''